source/Nevermore.Tests/CommandParameterValuesFixture.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using FluentAssertions;
+using Microsoft.Data.SqlClient;
+using NUnit.Framework;
+
+namespace Nevermore.Tests
+{
+    public class CommandParameterValuesFixture
+    {
+        [Test]
+        public void ShouldReplaceParametersWithExactMatch()
+        {
+            var parameters = new CommandParameterValues();
+            parameters.Add("someId", new[] { "id1", "id2" });
+            parameters.Add("someIdentifier", "value");
```

Comment:
Should we also test the reverse? where `someId` is just one value and `someIdentifier` is an array?